### ISSUE TITLE
[Build 1.1.1-7] omigen_py doesn't work:"failed to find configuration file"

### ISSUE THREAD

**Issue by Wayenyin** [Microsoft/omi-script-provider#29]
**ENV**
- CentOS 7 x64
- Oracle 7 x64
- SUSE 12 x64
- RHEL 7 x64
- Debian 7 x64, Debian 8 x64, Debain 9 x64
- Ubuntu 12 x64, Ubuntu 14 x64, Ubuntu16 x64

**Repos:**
1. Install omi version 1.3.0-105;
2. Install python script build: [\\\\redmond\wsscfs\OSTCData\Builds\omiscriptprovider\develop\1.1.1-7](url) ;
3. Currently copy customized python provider to /opt/omi/lib, it will be fixed [https://github.com/Microsoft/omi-script-provider/issues/22](https://github.com/Microsoft/omi-script-provider/issues/22);
4. Just copy provider MOF file of [xyz_frog](https://github.com/Microsoft/omi-script-provider/tree/master/samples/xyz_frog) to /root/ on Linux;
5. Execute omigen_py command but failed: “_/opt/omi/bin/omigen_py /root/schema.mof XYZ_Frog_”
    ```
    root@py64-cent7-t2  # /opt/omi/bin/omigen_py /root/schema.mof XYZ_Frog
    failed to find configuration file

    ~ 
    root@py64-cent7-t2  # /opt/omi/bin/omigen_py --help
    failed to find configuration file

    ```
**Expectly:** The omigen_py can generate mi_main.py and schema.py.
**Actual:** The omigen_py does not work.

**Comment by alinbalutoiu** [comment]
Seems to be related to Issue #31. @EMumau is omigen_py supposed to do anything if the OMI was not built from source?

**Comment by Wayenyin** [comment]
This issue is fixed on Python script provider version 1.1.1-14
